api/inventory/agents.proto

```diff
@@ -2,11 +2,29 @@ syntax = "proto3";
 
 package inventory;
 
+// It's used to handle error responses for all services in this file.
+// So this option adds default response into generated swagger file.
+// Then go-swagger generates reader for default response.
+// As default response server returns ErrorResponse.
+option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
+  responses: {
+    key: "default"
+    value: {
+      description: "An error response."
+      schema: {
+        json_schema: {
+          ref: ".inventory.ErrorResponse"
+        }
+      }
+    }
+  }
+};
```

Comment:
It looks like we can move that to `header.json`? Let's do so.

Reply:
Actually we can't, it generates response for each endpoint. AFAIK, Swagger 2.0 don't have feature to set global response for all endpoints.